flow-report/App.tsx

```diff
@@ -5,38 +5,38 @@
  */
 
 import {FunctionComponent} from 'preact';
-import {useState} from 'preact/hooks';
+import {Sidebar} from './Sidebar';
+import {useCurrentStep} from './hooks';
```

Comment:
let's try to avoid calling these "steps" if we can because for an eventual Muppet alignment "step" refers to a very specific, more granular user action.

Alternative suggestions:

- Just call it `useCurrentLhr`
- `section`
- `stage`
- `segment`
- `portion`

Like any of those?

Reply:
> Just call it `useCurrentLhr`

I'm thinking this one